index.js

```diff
@@ -41,7 +47,15 @@ function getAllMovieTitles() {}
  *  getHighestMetascore(movies);
  *  //> 96
  */
-function getHighestMetascore() {}
+function getHighestMetascore(movies) {
+  let myDefault = 0
+  for (let i = 0; i < movies.length; i ++){
+    if (movies[i].metascore > myDefault){
+      myDefault = movies[i].metascore
+    } return parseInt(myDefault)
```

Comment:
Take a look at our return and the scope its in - when during our loop is this returning?